### PR TITLE
Update Store.md

### PR DESCRIPTION
**Pull request by willnwhite**
Because I spent ages looking at middleware alternatives when I hadn't thought to do this: http://stackoverflow.com/questions/35305661/where-to-write-to-localstorage-in-a-redux-app.

### CODE REVIEW

**Comment by gaearon** [comment]
To be honest I don’t think this is particularly discoverable here.

In 95% of the cases `subscribe()` is used to update the UI (although higher level libraries like React Redux tend to abstract over its usage) so saying this is why it exists might give the wrong impression.

Maybe rewording it to something like

>It is most often used to update the UI in response to the state changes. Libraries like [React Redux](https://github.com/reactjs/react-redux) provide higher level abstractions on top of it but you can always use it directly, too. One common use case for it is to persist some parts of the current state into the `localStorage` on every change.

**Comment by willnwhite** [comment]
Yes, I agree about the discoverability. I've created a new PR for the gist code on the front page. I agree with your rewording for here.

**Comment by gaearon** [comment]
Closing in favor of #1620.
